bowtie/templates/report.html.j2

```diff
@@ -3,424 +3,437 @@
 
   .hover-details {
-      display: flex;
-      flex-direction: row;
-      justify-content: space-between;
-      gap: 0.7rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 0.7rem;
   }
 
   .hover-details span {
-      font-style: italic;
-      font-weight: 500;
-      color: rgb(112, 112, 112);
-      font-size: 0.8rem;
+    font-style: italic;
+    font-weight: 500;
+    color: rgb(112, 112, 112);
+    font-size: 0.8rem;
   }
-
-
 </style>
 
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Bowtie</title>
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
-  </head>
-
-  <body>
-    <nav class="navbar navbar-expand-lg sticky-top bg-light mb-4">
-      <div class="container-fluid">
-        <a class="navbar-brand" href="#">Bowtie</a>
-        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-          <span class="navbar-toggler-icon"></span>
-        </button>
-        <div class="collapse navbar-collapse" id="navbarNav">
-          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-            <li class="nav-item">
-              <a class="nav-link" href="#run-info">Run Info</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="#summary">Summary</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="#cases">Details</a>
-            </li>
-            {% if generate_dialect_navigation %}
-              <li class="nav-item">
-                <a class="nav-link" href="#"><b>Dialects:</b></a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="draft2020-12.html">2020-12</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="draft2019-09.html">2019-09</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="draft7.html">7</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="draft6.html">6</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="draft4.html">4</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="draft3.html">3</a>
-              </li>
-            {% endif %}
-          </ul>
-        </div>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Bowtie</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
You're changing (updating) the version here, and specifically to an alpha version -- I happen to be OK with that if it's necessary for whatever reason (even to the alpha version which would normally not be OK) but when submitting a PR you should definitely call that out specifically -- that you're updating the version -- and tell me the reviewer why/what made you do it. If it's not related to functionality in this PR then doing this change is similar to the formatting change in that it might be good to do but normally you shouldn't do it at the same time as the other changes, it makes things hard to review and hard to debug.

But yeah all that explanation aside, is there a reason it's needed?

Secondly, and also importantly, you dropped the `crossorigin="anonymous"` and `integrity` attributes both here and at the bottom in the other script tag. I have to assume the `integrity` change at least was a mistake, since there's no reason *not* to include it. The `crossorigin="anonymous"` attribute probably does no harm nor no benefit for this kind of thing, but still for both of those the same advice as above applies -- you should be super vocal about removing them if you did it on purpose, and if not, it's a good lesson to make sure you review the `git diff` carefully every time you submit a PR or change -- you may have had difficulty yourself doing so for the same reason a reviewer does/would because of the formatting changes, but still!

Reply:
## Why I changed cdn to alpha?
I was thinking about how to implement dark mode and light mode in this site with custom JavaScript and CSS,
And When going through the Bootstrap docs, found in v5.3.0 they are added this functionality. That I don't have to write custom css and js for these. I just simply use bootstrap class for this.
- Gained some trust, when saw some of their ( premium themes ) [Bootstrap Premium Themes]( https://themes.getbootstrap.com/) using these and in their home page the are showing the alpha version (not in corner or in small text).
![Screenshot_89](https://user-images.githubusercontent.com/92979541/231255644-8724abf5-9f66-4f64-a0ab-f4340b27dd8f.png)

Reply:
## Why I dropped crossorigin and integrity ?
I did not dropped those on purpose. I was doing those old fashioned way ( How I used to do before )
![cdns](https://user-images.githubusercontent.com/92979541/231256854-fa713777-b26a-4aa0-9ae2-7a77825d6a6b.png)
Copy the CDNs from here and paste in the href of <script> tag and in <link> tag for html and css.

After this comment I did some googling and found out
If you do not use the integrity attribute, it is possible for the file to be modified or replaced, which could cause security issues or break your website's functionality.